src/backend/columnar/Makefile

```diff
@@ -0,0 +1,35 @@
+safestringlib_srcdir = $(citus_abs_top_srcdir)/vendor/safestringlib
+MODULE_big = columnar
+
+OBJS = safeclib/mem_primitives_lib.o \
+       safeclib/memcpy_s.o \
+	   safeclib/safe_str_constraint.o \
+	   safeclib/safe_mem_constraint.o \
+	   safeclib/ignore_handler_s.o \
+	   safeclib/abort_handler_s.o \
+       columnar.o \
+       columnar_tableam.o \
+	   columnar_storage.o \
+	   columnar_compression.o \
+	   columnar_customscan.o \
+	   columnar_debug.o \
+	   columnar_metadata.o \
+	   columnar_reader.o \
+	   columnar_writer.o \
+	   mod.o \
+	   write_state_management.o 
+
+PG_CPPFLAGS += -I$(libpq_srcdir) -I$(safestringlib_srcdir)/include 
+
+DATA = $(wildcard sql/*--*.sql)
```

Comment:
Is this already being installed as part of the `generated_sql_files` in `distributed/Makefile`?

Reply:
Yes, I will remove that